lib/taskana-core/src/main/java/pro/taskana/mappings/WorkbasketMapper.java

```diff
@@ -150,6 +152,12 @@
         @Result(property = "orgLevel4", column = "ORG_LEVEL_4")})
     List<WorkbasketSummaryImpl> findAll();
 
+    @Select("<script>SELECT ID FROM WORKBASKET "
+        + "WHERE ID IN( <foreach item='item' collection='workbasketIds' separator=',' >#{item}</foreach> ) "
+        + "<if test=\"_databaseId == 'db2'\">with UR </if> "
+        + "</script>")
+    List<String> findExistingWorkbaskets(@Param("workbasketIds") List<String> workbasketIds);
```

Comment:
This might not be required. If it is, maybe it can be replaced by a "standard workbasket query"?